src/component/tooltip/tooltipMarkup.ts

```diff
@@ -278,20 +318,22 @@ const builderMap: { [key in TooltipMarkupBlockFragment['type']]: TooltipMarkupFr
             // It little weird if only value next to marker but far from marker.
             const valueCloseToMarker = !noMarker && noName;
 
+            const {nameStyle, valueStyle} = getTooltipTextStyle(toolTipTextStyle, renderMode);
+
             return renderMode === 'richText'
                 ? (
                     (noMarker ? '' : markerStr)
-                    + (noName ? '' : wrapInlineNameRichText(ctx, readableName))
+                    + (noName ? '' : wrapInlineNameRichText(ctx, readableName, nameStyle as RichTextStyle))
                     // Value has commas inside, so use ' ' as delimiter for multiple values.
                     + (noValue ? '' : wrapInlineValueRichText(
-                        ctx, readableValueList, valueAlignRight, valueCloseToMarker
+                        ctx, readableValueList, valueAlignRight, valueCloseToMarker, valueStyle as RichTextStyle
                     ))
                 )
                 : wrapBlockHTML(
                     (noMarker ? '' : markerStr)
-                    + (noName ? '' : wrapInlineNameHTML(readableName, !noMarker))
+                    + (noName ? '' : wrapInlineNameHTML(readableName, !noMarker, nameStyle as string))
                     + (noValue ? '' : wrapInlineValueHTML(
-                        readableValueList, valueAlignRight, valueCloseToMarker
+                        readableValueList, valueAlignRight, valueCloseToMarker, valueStyle as string
                     )),
                     topMarginForOuterGap
                 );
```

Comment:
I think here is an abuse of short hand if-else statement. It is inviting bugs to live inside and adds difficulty in debugging it.